### PR TITLE
feat: add PolygonShapeContent

### PR DESCRIPTION
**Pull request by ArneD**
Depending on PR #13 could be merged first as a minor version increase.

### CODE REVIEW

**Comment by CumpsD** [comment]
@yreynhout can you review this please?

File: test/Be.Vlaanderen.Basisregisters.Shaperon.Tests/ShapeBinaryWriterTests.cs [hunk 1]
```diff
@@ -26,15 +26,18 @@ public ShapeBinaryWriterTests()
                 customization.FromFactory(generator =>
                     {
                         var result = ShapeType.NullShape;
-                        switch (generator.Next() % 3)
+                        switch (generator.Next() % 4)
                         {
                             //case 0:
                             case 1:
-                                result = ShapeType.Point;
+                                result = ShapeType.PointM;
```
Comment (yreynhout) [inline test/Be.Vlaanderen.Basisregisters.Shaperon.Tests/ShapeBinaryWriterTests.cs]: Why PointM? PointM only makes sense as part of a PolyLineM. PointM is on its way out so not really that big a deal.

Comment (ArneD) [reply test/Be.Vlaanderen.Basisregisters.Shaperon.Tests/ShapeBinaryWriterTests.cs]: My bad. Added commit to fix.

**Pull request merged**